resources/deb/postinst.erb

```diff
@@ -2,21 +2,21 @@
 <%- get_services.each do |service| -%>
   # switch based on systemd vs systemv
   #
-  <%- if @platform.servicetype == "systemd" -%>
-if [ -z "$2" ]; then
-  systemctl enable <%= service.name %>.service >/dev/null || :
+if [ -d '/run/systemd/system' ] ; then
```

Comment:
Why not use `@platform.get_service_types.include?("systemd")` like you did for rpm?

Reply:
This is adding support for installing both sysv and systemd scripts for debian packages, so we need to switch based on what's actually running on the system